pax-compiler/src/parsing.rs

```diff
@@ -281,23 +281,25 @@ fn recurse_pratt_parse_to_string<'a>(
             },
             Rule::xo_symbol => {
                 symbolic_ids.borrow_mut().push(primary.as_str().to_string());
-                format!("{}",convert_symbolic_binding_from_paxel_to_ril(primary))
+                format!("({}).to_pax_any()",convert_symbolic_binding_from_paxel_to_ril(primary))
             },
             Rule::xo_tuple => {
                 let mut tuple = primary.into_inner();
                 let exp0 = tuple.next().unwrap();
                 let exp1 = tuple.next().unwrap();
                 let exp0 = recurse_pratt_parse_to_string( exp0.into_inner(), pratt_parser, Rc::clone(&symbolic_ids));
                 let exp1 = recurse_pratt_parse_to_string( exp1.into_inner(), pratt_parser, Rc::clone(&symbolic_ids));
-                format!("({}.into(),{}.into())", exp0, exp1)
+                let exp0 = exp0.trim_end_matches(".to_pax_any()");
```

Comment:
To avoid trimming we decided to divide up into two functions, one that wrapps the output in a pax any and one that does not, allowing the container cases to call the function that does not wrapp, instead of having to trim the output.